src/airport.h

```diff
@@ -124,6 +124,7 @@ enum class AirportBlock : uint8_t {
 	RunwayOut2       = 10, ///< @note re-uses #AirportBlock::TaxiwayBusy
 	HelipadGroup     = 13, ///< @note re-uses #AirportBlock::AirportEntrance
 	OutWay3          = 31,
+	Zeppeliner       = 32, ///< Block for the zeppeliner disaster vehicle.
```

Comment:
Maybe this 'special' block state with extra features should be next to `AirportClosed`, i.e. with a value of `62`. That way eventual future functional blocks can remain contiguous.